Failure in TestSendDisembargo/SendQueuedResultToCaller

```
--- FAIL: TestSendDisembargo (0.01s)
    --- FAIL: TestSendDisembargo/SendQueuedResultToCaller (0.01s)
Error:         level1_test.go:285: Received finish message; want disembargo
Error:         level0_test.go:2126: conn error: rpc: send release: send message: context canceled
```

_Originally posted by @lthibault in https://github.com/capnproto/go-capnproto2/issues/424#issuecomment-1382569423_

--- a/rpc/level1_test.go
+++ b/rpc/level1_test.go
@@ -249,7 +249,7 @@ func testSendDisembargo(t *testing.T, sendPrimeTo rpccp.Call_sendResultsTo_Which
 		err = pogs.Insert(rpccp.Message_TypeID, capnp.Struct(outMsg.Message), &rpcMessage{
 			Which: rpccp.Message_Which_return,
 			Return: &rpcReturn{
-				AnswerID: bootQID,
+				AnswerID: qidA,
 				Which:    rpccp.Return_Which_results,
 				Results: &rpcPayload{
 					Content: results.ToPtr(),

